libnmstate/nm/sriov.py

```diff
@@ -148,23 +151,34 @@ def _has_sriov_capability(ifname):
 
 def get_info(device):
     """
-    Provide the current active SR-IOV live configuration for a device
+    Provide the current active SR-IOV total-vfs runtime value and the live
+    configuration for each VF for a device.
     """
     info = {}
+    sriov_config = {}
+
+    ifname = device.get_iface()
+    numvf_path = f"/sys/class/net/{ifname}/device/sriov_numvfs"
+    if os.path.exists(numvf_path):
+        try:
+            with open(numvf_path) as f:
+                sriov_config[Ethernet.SRIOV.TOTAL_VFS] = int(f.read())
+        except FileNotFoundError:
```

Comment:
Instead of checking if the path exists, return info here in the except block. It is odd to catch this specific exception here if he code first checks if the file exists and does not treat it as an error.

Reply:
Done! Thanks :-)